plugins/modules/elb_classic_lb_info.py

```diff
@@ -190,21 +387,53 @@ def get_lb(connection, load_balancer_name):
         return []
 
 
-def get_lb_attributes(connection, load_balancer_name):
+def get_lb_attributes(connection: Any, load_balancer_name: str) -> Dict:
+    """
+    Retrieves attributes of specific Elastic Load Balancer (ELB) by name.
+
+    Parameters:
+      connection (boto3.client): The Boto3 ELB client object.
+      load_balancer_name (str): Name of the ELB to gather information about.
+
+    Returns:
+      A dictionary with detailed information of the attributes of specified ELB.
+    """
     attributes = connection.describe_load_balancer_attributes(aws_retry=True, LoadBalancerName=load_balancer_name).get(
```

Comment:
I also think you can import `describe_load_balancer_attributes()` from https://github.com/ansible-collections/amazon.aws/blob/main/plugins/module_utils/elb_utils.py#L86

Reply:
Had seen elb_utils.py, but from my understanding all those API calls and client used in that file are for ELBv2 https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/elbv2/client/describe_load_balancers.html
and do not work for classic lb (ELB) https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/elb/client/describe_load_balancers.html

Reply:
Ah ok, I saw the file called elb_* and I thought we could use it. Thanks.

Reply:
I will mark the suggestions since they do not apply for the moment.